test/db/cmd/cmd_help

```diff
@@ -242,4 +242,36 @@ EOF
 EXPECT=<<EOF
 0x116e
 EOF
-RUN
\ No newline at end of file
+RUN
+
+NAME=recursive help
+FILE=-
+CMDS=<<EOF
+?* zi
+EOF
+EXPECT=<<EOF
+Usage: [0mzi[0m[j*qi][0m   # Show zignatures matching information[0m
```

Comment:
I think the `Usage` line should not be present because all commands are already listed anyway and it makes the output a bit messy IMO.